test/method/testmoments.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Test the Moments method in cclib"""
+
+import sys
+import unittest
+from unittest import mock
+
+import numpy as np
+from numpy.testing import assert_equal, assert_almost_equal
+
+from cclib.method import Moments
+
+
+class TestIdealizedInputs(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.linear_dipole_attrs = {
+            'atomcoords': np.array([[[-1, 0, 0], [ 1,  0, 0]]]),
+            'atomcharges': {'mulliken': [-1, 1]},
+            'atomnos': [1, 1],
+        }
+    
+    @mock.patch('cclib.parser.ccData', spec=True)
```

Comment:
Do we really require a mock for this? Isn't it easy to just set up a real ccData object?